index.js

```diff
@@ -52,7 +54,7 @@ client.on("message", (message) => {
 		var tomb = new can.Image();
 		// set variables for the image URLs. Tombstone and user pfp
 		img.src = user.avatarURL;
-		tomb.src = "https://seebeyond.space/snippets/assets/bingo/tomb.png";
+    tomb.src = `https://seebeyond.space/snippets/assets/bingo/${tombstone_type}.png`; // TODO: add fallback to tomb OR throw "Tomb not recognized" error
 		// when user pfp (img) is done loading

```

Comment:
and if Tomb Not Recognized error is thrown, continue sending the message with the default Tombstone